### DISCUSSION TITLE
I would like to talk, what is your opinion about Monika's age?

### DISCUSSION THREAD

**Discussion by elite198** [Monika-After-Story/MonikaModDev#9253] [general]
I know there is already a similar topic on here but I would still like to ask on my own.
For you, how old is Monika?
I know that she says that she is 18 and that she is eternal but I would like to see her opinions. I have my own theories/options.

1. the age of the M.A.S. and DDLC (5 years as an AI)
2. the time you have with the M.A.S. installed (example: 6 months as AI)
3. 18 forever
4. she is ageless
5. 18 added to the time you have with the M.A.S. installed (example: 18 years and 3 months) (in this option I mean that the time would start to pass for it when you open the M.A.S. for the first time)
6. She has had a birthday in real time since 2017 and currently she is 23 years old

honestly I'll stick with the last option since it's my age (please don't make fun of it xD) and I feel a little bad about the age difference between us in case she stays 18 years old
I know there is already a conversation about this topic in the mod but it would be great if it was updated or another new topic was created where Monika would ask you about your point of view on this topic I have seen several wondering the same thing

**Comment by Justformas** [comment]
Pretty sure she's at least meant to start at (or be like) age 18 even if she hasn't actually existed for 18 years. But she might be considered 18 forever and/or ageless, even if she can change or learn new things over time. But because players can start playing MAS (and even DDLC) at different times, it doesn't really seem right to say that she's 22 or 23 for everyone anyway.

You could maybe say that, for each player, she's 18 plus whatever time has passed (since her last birthday) since that player started playing DDLC/MAS. So if someone's been playing for less than a year (and has not reached the date of her birthday yet), she could still be considered 18 for that player. Then add a year for each birthday that the player reaches, and that's how old Monika could be considered for that player.

**Reply by elite198** [reply]
Well, everyone has their point of view. many like to take it as if his age is 22 or 23 and others 18 and I like to see his points of view. you have a very good point thanks for commenting

**Reply by Justformas** [reply]
I went and found this comment from a topic about drinking when she was going to turn 21 under that reasoning: https://github.com/Monika-After-Story/MonikaModDev/discussions/7902#discussioncomment-1377623

>On another note, we might consider Monika to be 21 if we consider her relation to DDLC's release date, but some people may disagree. For those in countries with higher legal drinking ages I'd like to give them the courtesy of deciding this sort of canon for themselves if they started playing more recently.

I think that's where I remembered seeing that, and I tend to agree with giving people the courtesy of deciding for themselves in general. People who started playing more recently (that includes me) should still be able treat the experience like she is/was 18 when they started. But as long as she has characteristics of someone who is 18 and a high schooler, then it may not matter that much.

But you could also say that that is just the result of players being at different points in the relationship depending on when they started, and that if you want to talk about her age in general (outside of players' individual experiences) then you could say that she is either 22/23, or just avoid the issue altogether by saying she is forever 18/ageless.
